lib/js/__tests__/Aeson_decode_test.js

```diff
@@ -130,6 +130,15 @@ Jest.describe("float", (function (param) {
         Jest.test("float", (function (param) {
                 return Jest.Expect.toEqual(Jest.Expect.expect(Aeson_decode.$$float(Aeson_encode.$$float(Number("-Infinity")))), Number("-Infinity"));
               }));
+        Jest.test("float", (function (param) {
```

Comment:
![Screenshot(47)](https://github.com/user-attachments/assets/d364814c-e826-4be2-bd59-be6e0735f109)
^ I think only test for `+Infinity` was missing

Reply:
They are slightly different tests, one encodes a float directly, the other encodes a string then decodes a float:
```rescript
    test("float", () =>
      expect(Aeson.Decode.float(Aeson.Encode.float(Js.Float.fromString("Infinity"))))->toEqual(
        Js.Float.fromString("Infinity"),
      )
    )
    test("float", () =>
      expect(Aeson.Decode.float(Aeson.Encode.string("Infinity")))->toEqual(
        Js.Float.fromString("Infinity"),
      )
    )
```